go/service/chat_local_test.go

```diff
@@ -65,6 +65,28 @@ func TestNewConversationLocal(t *testing.T) {
 	}
 }
 
+func TestNewChatConversationLocalTwice(t *testing.T) {
+	ctc := makeChatTestContext(t, "NewConversationLocal")
+	defer ctc.tc.Cleanup()
+
+	mustCreateConversationForTest(t, ctc, chat1.TopicType_CHAT, "t_alice")
+
+	if _, err := ctc.h.NewConversationLocal(context.Background(), chat1.ConversationInfoLocal{
+		TlfName:   "t_alice," + ctc.world.me.Username,
+		TopicType: chat1.TopicType_CHAT,
+	}); err == nil {
+		t.Fatalf("creating CHAT conversation twice did not fail")
```

Comment:
Wasn't the point that NewConversation would return an existing conversation, not an error?